examples/platforms/efr32/radio.c

```diff
@@ -102,6 +104,8 @@ typedef enum
     ENERGY_SCAN_MODE_ASYNC
 } energyScanMode;
 
+RAIL_Handle_t gRailHandle;
```

Comment:
When multiband mode is enabled (`RADIO_CONFIG_2P4GHZ_OQPSK_SUPPORT` and `RADIO_CONFIG_915MHZ_OQPSK_SUPPORT` enabled), `efr32RailConfigInit()` is called twice. Which RAIL handle should we be using globally? Both?

Reply:
This also should be initialised to NULL so as a basic sanity check, external users of the global variable can check if it has really been initialised, rather than working with a dangling pointer.